apiary.apib

```diff
@@ -769,6 +769,49 @@ This operations is available only for organization administrators.
                 "sharing": "organization"
             }
 
+### Share Bucket To Project [POST /v2/storage/buckets/{bucket_id}/share-to-projects]
+[Enables sharing](https://help.keboola.com/storage/buckets/sharing/) or **update sharing** for this bucket within the organization.
+Any organization member will be able to link to this bucket from specific project within the organization.
```

Comment:
Hi, @pivnicek may I ask you for check this. Thanks

Reply:
Hi, I would say:
```
[Enables sharing](https://help.keboola.com/storage/buckets/sharing/) or **updates sharing** for this bucket to a specific set of projects.
Any user who is a member of one of the targeted projects will be able to link to this bucket from that specific project.
```
But I have 2 questions:
- 1 Do I have to be an organization admin to make a link, or can I do it if I'm just an ordinary project member?
- 2 Can I share the bucket to a project that is not in the organization?

Reply:
Thanks :) 

1. For enable sharing have to be an organization admin how is it in other sharing like before (organization, organization-project) and you can link shared bucket to your project as an ordinary project member. 

2.  No, you can share the bucket only for project in the same organization.

Reply:
Thanks, ok, I understand.  In that case:
```
[Enables sharing](https://help.keboola.com/storage/buckets/sharing/) or **updates sharing** for this bucket to a specific set of projects in the organization.
Any user who is a member of one of the targeted projects will be able to link to this bucket from that specific project.
This operations is available only for organization administrators.
```